doc/operator-monitoring-resources.md

```diff
@@ -27,25 +25,114 @@ spec:
     enabled: true
 ```
 
+NOTE: All monitoring resources will be created by the operator using *Create only* reconcilliation policy. That means that PrometheusRules and GrafanaDashboards objects can be updated preventing the operator to revert the changes back. This policy allows to tune, for instance, the alert thresholds, to your needs.
+
 ## Monitored components
 
 * Kubernetes resources at namespace level where 3scale is installed
 * Apicast Staging
 * Apicast Production
 * 3scale Backend worker
 * 3scale Backend listener
-* System sidekiq
+* System
 * Zync
 * Zync-que
 
-## Exposing monitoring resources
+## Monitoring stack
+
+3scale monitoring is leveraged by [prometheus](https://prometheus.io/) and [grafana](https://grafana.com/) monitoring solutions. They need to be up and running in the cluster and configured to watch for monitoring resources.
+
+Monitoring stack setup and running is beyond 3scale operator reponsabilities.
+There are many ways to provide the required monitoring stack. Few examples are given:
+
+* Openshift new proposal [User Workload Monitoring](https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/user-workload-monitoring.md)
+
+* [kube-prometheus](https://github.com/coreos/kube-prometheus): This repository collects Kubernetes manifests, Grafana dashboards, and Prometheus rules combined with documentation and scripts to provide easy to operate end-to-end Kubernetes cluster monitoring with Prometheus using the Prometheus Operator.
+
+* For devtesting purposes only, *quickstart steps* to deploy minimum required monitoring stack in [monitoring-stack-deployment](monitoring-stack-deployment/README.md)
```

Comment:
I would remark something like, this dev testing examples make services to be published to internet without any kind of auth, which is something not recommended at all (otherwise anyone can access to your prometheus instance for example).

In our dev clusters we configure authentication to be managed by Openshift (so htpassword or github)

Reply:
Added.